tests/url_utils.test.ts

```diff
@@ -135,26 +161,26 @@ describe("Loading + saving from URL query strings", () => {
   it("Enforces min/max on range and thresholds", () => {
     const originalParams: Partial<UrlParams> = {
       thresholds: [
-        { featureName: "feature1", units: "m", min: 1, max: 0 },
-        { featureName: "feature2", units: "m", min: 12, max: -34 },
-        { featureName: "feature3", units: "m", min: 0.5, max: 0.25 },
+        { featureName: "feature1", units: "m", type: ThresholdType.NUMERIC, min: 1, max: 0 },
+        { featureName: "feature2", units: "m", type: ThresholdType.NUMERIC, min: 12, max: -34 },
+        { featureName: "feature3", units: "m", type: ThresholdType.NUMERIC, min: 0.5, max: 0.25 },
       ],
       range: [1, 0],
     };
     const queryString = paramsToUrlQueryString(originalParams);
     const parsedParams = loadParamsFromUrlQueryString(queryString);
 
     expect(parsedParams.thresholds).deep.equals([
-      { featureName: "feature1", units: "m", min: 0, max: 1 },
-      { featureName: "feature2", units: "m", min: -34, max: 12 },
-      { featureName: "feature3", units: "m", min: 0.25, max: 0.5 },
+      { featureName: "feature1", units: "m", type: ThresholdType.NUMERIC, min: 0, max: 1 },
+      { featureName: "feature2", units: "m", type: ThresholdType.NUMERIC, min: -34, max: 12 },
+      { featureName: "feature3", units: "m", type: ThresholdType.NUMERIC, min: 0.25, max: 0.5 },
```

Comment:
Minor: I think it would be good if these tests that go to and from urlquerystring also tested against the expected generated query string. ( expect queryString to equal whatever )

Reply:
Made this change! These are ALL going to break when I update how filter URLs are stored later 😅